Alignment/APEEstimation/plugins/createDefaultApeTree.cc

```diff
@@ -0,0 +1,474 @@
+
```

Comment:
hi @mteroerd - ApeTreeCreateDefault.cc would better fit CMS conventions (starting classes with capital letters) and the other plugins in this package